docs/guides/key-manager/give-permissions.md

```diff
@@ -90,28 +91,27 @@ We also need to add the `beneficiaryAddress` inside the `AddressPermissions[]` A
 
 ```js
 // step 2.2 - encode the data key-value pairs of the permissions to be set for the beneficiary address
-const beneficiaryAddress = "0xcafecafecafecafecafecafecafecafecafecafe"; // EOA address of an exemplary person
```

Comment:
@Hugoo @fhildeb we should add a `.prettierrc` for Markdown or some config in the docs.lukso.tech to prevent these diffs due to local prettier configs between team members.